### ISSUE TITLE
No signal from PS4 Pro on 9.00 - Rpi3 b+

### ISSUE THREAD

**Issue by Pasqui1978** [stooged/PI-Pwn#11]
![IMG20240510183319](https://github.com/stooged/PI-Pwn/assets/104250169/f310f9d7-e154-4af6-96fa-9e1a80283344)
![IMG20240510173727](https://github.com/stooged/PI-Pwn/assets/104250169/b26a279e-9acd-4fe5-890c-409cca41b71e)
Formatted and remake with latest release.

**Comment by Pasqui1978** [comment]
![IMG20240510184920](https://github.com/stooged/PI-Pwn/assets/104250169/198f74d1-e6ec-4d46-92c3-25c0eb09dacd)

**Comment by jmarkk** [comment]
same issue on my side

**Comment by rahan-atome** [comment]
EXACT same issue for me on my ps4 pro!
fw 11, Pi 3b+

**Comment by Pasqui1978** [comment]
> same issue on my side 
> fw 11, Raspberry Pi Zero W with usb ethernet

Different firmware, different Rpi, different cable... not the same.

**Comment by rahan-atome** [comment]
> > same issue on my side
> > fw 11, Raspberry Pi Zero W with usb ethernet
> 
> Different firmware, different Rpi, different cable... not the same.

Résultats whas same : 
![IMG_20240510_182628](https://github.com/stooged/PI-Pwn/assets/75158317/58d5c21d-a23d-4fd4-8a39-d8604550e829)
![IMG_20240510_221605](https://github.com/stooged/PI-Pwn/assets/75158317/6c075b37-ee31-4fbf-9098-a3b2cef5dcc3)

**Comment by sangimed** [comment]
Same issue here.
# Specs:
- raspberry pi 3 b+
- PS4 Slim 1Tb
- Ethernet Cable
- FW 11.00

**Comment by Doyle4** [comment]
https://www.youtube.com/watch?v=yvivDdRirLA&ab_channel=MODDEDWARFARE

Could some of you guys.

**Comment by lotech** [comment]
Also having the same issue - all is looking good but no luck.
Having a PS4 Pro seams to be part of the issue.

**Comment by stooged** [comment]
my ps4 pro is on 9.00 and it works its a 7102b

**Comment by shoyrock** [comment]
I'm on firmware 9.00
I want to test the new exploit With my Raspberry Pie 4 B. I waited for about 5+ minutes for the PS4 to be pown. I left to get something to eat because of how long it took. After I came back, the PS4 was jailbroken, and everything seemed to be ok. I powed to the PS4 and rebooted to see how long it took for the exploit to execute. After waiting some minutes, the PS4 turned off, so I turned it on again. I came back up and started to rerun the exploit. The PS4 shut off. After I turned it on again, the screen was black. I haven’t been able to get an image since. I  booted into the recovery option and changed the resolution to fix the problem.
